docs/src/api/class-chromiumbrowser.md

```diff
@@ -39,6 +39,7 @@ const { chromium } = require('playwright');
 ```
 
 ## async method: ChromiumBrowser.newBrowserCDPSession
+* langs: js, python
```

Comment:
The entire class is marked as js-only:
```
# class: ChromiumBrowser
* langs: js	* langs: js
```
this is usually sufficient to filter out all child entities too.

Reply:
You're right. Not sure why I had it anymore. Removed it. Also, I changed the way I fixed the problem by removing the reference in the paragraph for java and csharp which is probably the _better_ way of handling it...